docs/best-practices/crate-node.txt

```diff
@@ -0,0 +1,460 @@
+.. highlight:: psql
+.. _recover:
+
+====================================
+Troubleshooting with Crate-Node Tool
+====================================
+
+The crate-node command enables you to perform certain unsafe operations
+on a node that are only possible while it is shut down. This command allows you
+to adjust the role of a node and may be able to recover some data after a disaster.
+
+There are some rare scenarios where the cluster reaches a state where it cannot
+recover anymore by itself and external help is required to bring it back live.
```

Comment:
I think we can be more specific here and in the next paragraph.

E.g. along the lines of `If a CrateDB cluster unrecoverably looses the majority of the master eligible nodes, the `crate-node` CLI provides a way to utilize the state of the remaining master nodes to form a new cluster... `